sdk/src/Services/S3/Custom/Model/AbortMultipartUploadRequest.cs

```diff
@@ -145,9 +145,14 @@ internal bool IsSetExpectedBucketOwner()
         /// </summary>
         /// <remarks>
         /// This property will be used as part of the resource path of the HTTP request. In .NET the System.Uri class
-        /// is used to construct the uri for the request. The System.Uri class will canonicalize the uri string by compacting characters like "..". /// For example an object key of "foo/../bar/file.txt" will be transformed into "bar/file.txt" because the ".." 
-        /// is interpreted as use parent directory. For further information view the documentation for 
-        /// the Uri class: https://docs.microsoft.com/en-us/dotnet/api/system.uri
+        /// is used to construct the uri for the request. The System.Uri class will canonicalize the uri string by compacting characters like "..". 
+        /// For example an object key of "foo/../bar/file.txt" will be transformed into "bar/file.txt" because the ".." 
+        /// is interpreted as use parent directory. 
+        /// 
```

Comment:
`<para>...</para>` should be used because empty line brakes like this will not actually cause paragraph output where intended (IntelliSense, docs) in all of these duplicated blocks.

Reply:
Fixed